- [ ] Inventory what is here

Chards79 opened the pull request:
- [ ] Implement the CPU constructor
- [ ] Add RAM functions 
am_read() and 
am_write()
- [ ] Implement the core of 
un()
- [ ] Implement the HLT instruction handler
- [ ] Add the LDI instruction
- [ ] Add the PRN instruction

Mister-Corn commented on ls8/cpu.c:
```diff
@@ -63,5 +63,19 @@ void cpu_init(struct cpu *cpu)
+
+for (int i = 0; i <= 255; i++) {
+  cpu->ram[i] = 0;
 }
```
As an alternative to `for` loops, you can use [`memset`](https://www.tutorialspoint.com/c_standard_library/c_function_memset.htm) to set all the values to zero.

Mister-Corn commented on ls8/cpu.h:
```diff
@@ -2,13 +2,27 @@
+   // hold up what are you doing? O_O lol you said to do it as an array and then I wanted to iterate over each one to make them 0
+   // in the struct, you declare an array. you ain't doin stuff to it in the struct definition. or are you trying to null everything?  yeah beccause this
+   // which is still not in the struct definition. 
+   // the struct simply defines the structure of whatever you're making. the actual manipulation of the values it stores does not happen here
```
:arrow_double_up:

Chards79 committed:
- [ ] Inventory what is here
- [ ] Implement the CPU constructor
- [ ] Add RAM functions 
am_read() and 
am_write()
- [ ] Implement the core of 
un()
- [ ] Implement the HLT instruction handler
- [ ] Add the LDI instruction
- [ ] Add the PRN instruction
